SSE_PROTOCOL_V2.md

```
@@ -57,8 +58,45 @@ data: <json>
 }
 ```
 
+- `seq`는 SSE 프레임 단위로 1씩 증가합니다.
+- `id`는 `run_id:seq` 포맷이며 클라이언트 재연결을 위한 기준으로 사용할 수 있습니다.
+
 ## 이벤트 taxonomy (현재 구현)
 
+## 이벤트/데이터 형식 표
+
+### 세션/런
+| event | data 필드 | 설명 | 예시 payload |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)